packages/projects-docs/pages/learn/repositories/getting-started/git-workflow.mdx

```diff
@@ -8,6 +8,16 @@ import { Callout } from 'nextra-theme-docs'
 
 # Git workflow
 
+## Terminal
+
+Working locally you where likely using the terminal to interact with Git. You can still use your terminal, with one exception: `git checkout`.
+
+Since each branch on CodeSandbox is its own MicroVM and environment, we will rather navigate you to the respective MicroVM when doing a `git checkout`. Normally this is just another way of navigating to a branch, but if you

```

Comment:
```suggestion
Each branch runs on its own virtual machine (VM). So instead of checking out a branch within the same VM, when you change a branch on CodeSandbox, we will navigate you to the branch running on its respective VM. 

However, if you choose to change branch from the terminal using `git checkout`, you will need to conform to the following

```